Tests/MapboxNavigationTests/RouteControllerSnapshotTests.swift

```diff
@@ -49,19 +47,19 @@ class RouteControllerSnapshotTests: FBSnapshotTestCase {
         let routeController = RouteController(along: route, routeIndex: 0, options: equivalentRouteOptions, dataSource: self)
         locationManager.delegate = routeController
         
-        var snappedLocations = [CLLocation]()
+        var snappedLocations = Set<CLLocation>()
```

Comment:
I suspect we’d still want to verify that the locations are recorded in the correct order, so an array would be more appropriate than a set.

Reply:
Thanks for the suggestion, I'll change the Set to array to verify the sequence.